srtcore/srt.h

```diff
@@ -33,7 +33,7 @@ written by
 //if compiling with MinGW, it only works on XP or above
 //use -D_WIN32_WINNT=0x0501
 
-
+#ifndef SRT_API
```

Comment:
I think this whole part should be removed from this file and put into `platform_sys.h` file where it belongs.

The definition of `SRT_API` should still stay in this file, but then this symbol shall not be used anywhere else in the SRT code than this exactly file.